### ISSUE TITLE
Remarques sur le catalogue d'objets vprojet_2023-11_Classes manquantes_Dimensions parcellaires

### ISSUE THREAD

**Issue by ynakhimo** [cnigfr/structuration-reglement-urbanisme#20]
## Dimensions parcellaires

Parmi les conditions qui touchent aux caractéristiques de la parcelle, on trouve actuellement les classes **SurfaceTerrain** et **RouteBordante** (renommée **VoirieBordante**). Or, il manque d'autres dimensions parcellaires, telles que la largeur et la profondeur, qui peuvent changer la contrainte à appliquer (soit-il en article 7, qui rend la règle moins contraignante pour des parcelles étroites, ou en article 13, qui prévoit des accommodements en matière de besoins en pleine terre pour des parcelles peu profondes).

On pourrait ajouter deux classes supplémentaires qui héritent de la **ConditionUnitaire**, voire regrouper le tout dans une seule classe Dimensions. 

p.s. Ces dimensions peuvent d'ailleurs avoir une définition complexe et spécifique, surtout dans des PLUi, e.g. [Plaine Commune](https://plainecommune.fr/fileadmin/user_upload/Portail_Plaine_Commune/LA_DOC/PROJET_DE_TERRITOIRE/PLUI/PLUi_Exutoire/TOME_4-REGLEMENT_ECRIT_ET_GRAPHIQUE/TOME_4-REGLEMENT_ECRIT/200057867_4-1-1_Partie1_Definitions_et_dispositions_generales.pdf#page=82).

**Comment by GT-CNIG-DDU** [comment]
Mea culpa @ynakhimo, j'ai oublié lors du SG6 ce matin que le modèle présente actuellement une classe `SurfaceParcelle` (avec les deux attributs SurfaceParcelle.`aireMin` et SurfaceParcelle.`aireMax`), ce qui militerait effectivement pour regrouper les informations, et si possible au niveau de cette classe.

Une solution ne consiste-t-elle pas à renommer cette classe en `DimensionsParcelle` (ou de façon plus générale, si nécessaire, `CaracteristiquesParcelle`) avec tous les attributs nécessaires (`aireMin`, `aireMax`, `largeur`, `profondeur`, etc. etc.) ?

Comment caractériser la caractéristique "étroite" ? S'agit-il d'un résultat déduit des différents attributs ci-dessus, ou faut-il plutôt prévoir un attribut etroit avec valeurs oui/non ? (on nommerait alors plutôt cette classe `CaracteristiquesParcelle`...)

**Comment by alisonlenain** [comment]
A présenter lors de prochaine réunion pour étudier les différentes options de modélisation.

**Comment by mbrasebin** [comment]
Pour le travail que nous avions fait en Ile de France, l'étroitesse avait été considéré en prenant en compte la longueur de la limite séparative donnant sur la voirie. 

Mais effectivement, je pense que la proposition d'avoir une classe de dimension de parcelle avec la possibilité d'activer les différentes contraintes est la plus intéressante. Par contre, est-ce que cette notion de largeur de parcelle est définie universellement ?

**Comment by alisonlenain** [comment]
Renommer la classe par DimensionParcelle.
Ajout des attributs « largeur » et « profondeur » (Type: Real). 
Reprendre les illustrations de profondeur mise à disposition dans  e.g. [Plaine Commune](https://plainecommune.fr/fileadmin/user_upload/Portail_Plaine_Commune/LA_DOC/PROJET_DE_TERRITOIRE/PLUI/PLUi_Exutoire/TOME_4-REGLEMENT_ECRIT_ET_GRAPHIQUE/TOME_4-REGLEMENT_ECRIT/200057867_4-1-1_Partie1_Definitions_et_dispositions_generales.pdf#page=82).

**Comment by alisonlenain** [comment]
Modifications validées avec le groupe à inscrire dans le standard

**Comment by alisonlenain** [comment]
Rajout de l'attribut "profondeurParcelle" dans la classe DimensionParcelle afin de prendre en compte l'ensemble de ces cas apparaissant dans les PLUs. Buildrz donnera des exemples afin de créer une définition de la profondeur s'appliquant à la parcelle.

**Comment by GT-CNIG-DDU** [comment]
@ynakhimo : en attente d'une "définition de la profondeur s’appliquant à la parcelle afin de couvrir les cas qui ne concernent pas les bandes de constructibilité."

**Comment by aurore7** [comment]
Modification de la définition de l’attribut profondeur en « Profondeur minimale ou maximale de la parcelle en mètres » avec possibilité de l’un des 2 champs vides [0-1] ?

**Comment by GT-CNIG-DDU** [comment]
Oui, si j'ai bien compris on remplace le champ actuel par deux champs (profondeur min de la parcelle et profondeur max de la parcelle)
